src/components/LiquidityMining/Create/Steps/PairAndReward/AssetSelector/index.tsx

```diff
@@ -1,77 +1,249 @@
-        <Card selectable onClick={onClick}>
-          <Flex height="26px" width="100%" justifyContent="center" alignItems="center">
-            <Box mr={currency0 ? '8px' : '0'}>
-              <AssetLogo currency0={currency0} currency1={currency1} />
-            </Box>
-            <Box>
-              <TYPE.body
-                lineHeight="20px"
-                color="white"
-                data-testid={title.toLocaleLowerCase().replace(' ', '-') + '-select'}
+    <Flex flexDirection={'column'} key={index}>
+      <SmoothGradientCard
+        selectable
+        isToken={true}
+        active={currency0 !== undefined}
+        paddingBottom={
+          isReward && index === 0 && currency0 === undefined ? '45px' : isReward ? '30px' : '28px' + '!important'
+        }
+        width={'162px'}
+        flexDirection={'column-reverse'}
+        height={isReward ? '192px' : '150px'}
+        onClick={onClick}
+      >
+        {isReward && currency0 && (
+          <RelativeDismiss
+            onClick={event => {
+              event.stopPropagation()
+              if (isReward) handleDismiss()
+            }}
+          />
+        )}
+
+        <Flex width="100%" justifyContent="center" alignSelf="end">
+          <AssetLogo campaingType={campaingType} currency0={currency0} currency1={currency1} />
+          <Flex flexDirection={'column'}>
+            {isReward && currency0 && handleUserInput ? (
+              <RelativeContainer>
+                <StyledNumericalInput
+                  style={{
+                    width: `${
+                      rawAmount && rawAmount.length > 16 ? '150' : rawAmount ? 17 + rawAmount?.length * 8 : 25
+                    }px`,
+                  }}
+                  onClick={event => event.stopPropagation()}
+                  placeholder="0"
+                  value={rawAmount ? rawAmount : ''}
+                  onUserInput={handleUserInput}
+                />
+              </RelativeContainer>
+            ) : (
+              //deleted data-testid={title.toLocaleLowerCase().replace(' ', '-') + '-select'}
+              <TYPE.largeHeader
+                letterSpacing={'0.08em'}
+                marginBottom={'4px'}
+                lineHeight="22px"
+                color={customAssetTitle || !tokenName ? '#8C83C0' : 'lightPurple'}
+                fontSize={13}
+              >
+                {customAssetTitle && !tokenName ? customAssetTitle : assetTitle}
+              </TYPE.largeHeader>
+            )}
+            {tokenName && (
+              <TYPE.small
+                letterSpacing={'0.05em'}
+                color="purple3"
+                fontSize={10}
+                fontWeight="600"
+                lineHeight={isReward ? '16px' : '12px'}
               >
-                {assetTitle || 'Click to select'}
-              </TYPE.body>
-            </Box>
+                {tokenName}
+                {isReward && (
+                  <span style={{ color: 'white' }}>
```

Comment:
I don't like this, this can be a styled-component

Reply:
this is where i think also inline style is okay don't agree that one property justifies creating component